docs/references/javascript/organization/domains.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Organization Domains
+description: These are all methods on the Organization class that allow you to manage the domains of an organization.
+---
+
+import { Tables } from "@/components/Table";
+
+# Organization Domains
+
+These are all methods on [the Organization class](/docs/references/javascript/organization/organization) that allow you to manage the domains of an organization.
+
+## `getDomains()`
+
+```typescript
+function getDomains(params?: GetDomainsParams): Promise<PaginatedResources<OrganizationDomain>>;
+```
+
+### `GetDomainsParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | number | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
+| `pageSize?` | number | A number that indicates the maximum number of results that should be returned for a specific page. |
+| `enrollmentMode?` | `'manual_invitation' \| 'automatic_invitation' \| 'automatic_suggestion'` | The mode in which a domain should operate under |
+
+### Returns
+
+<Tables
+    headings={["Type", "Description"]}
+    headingsMeta={[{}, {maxWidth: "300px"}]}
+    rows={[
+        {cells: [
+                <code>Promise{"<"}<a href="/docs/references/javascript/types/paginated-response">PaginatedResponse</a>{"<"}<a href="/docs/references/javascript/organization-domain">OrganizationDomain</a>{">"}</code>,
+                <>This method returns a <code>Promise</code> which resolves with a list of <code><a href="/docs/references/javascript/organization-domain">OrganizationDomain</a></code> objects.</>,
+            ]},
+    ]}
+/>
+
+
+## `getDomain()`
+
+```typescript
+function getDomain(params: GetDomainParams): Promise<OrganizationDomain>;
+```
+
+Creates and sends an invitation to the target email address for becoming a member with the role passed on the  function parameters.
```

Comment:
The method is getDomain() but takes about creating a domain? Is there a `createDomain()` method missing??

```suggestion
Creates and sends an invitation to the target email address for becoming a member with the role passed on the function parameters.
```